render/node.js

```diff
@@ -1,5 +1,5 @@
 function Node(tag, key, attrs, children, text, dom) {
-	return {tag: tag, key: key, attrs: attrs, children: children, text: text, dom: dom, domSize: undefined, state: {}, events: undefined, instance: undefined}
+	return {tag: tag, key: key, attrs: attrs, children: children, text: text, dom: dom, domSize: undefined, state: undefined, events: undefined, instance: undefined}
```

Comment:
I decided to lazily construct the component state for mainly consistency reasons (classes create their own object, anyways).